tests/reducers/myActivitiesReducer.test.js

```diff
@@ -0,0 +1,56 @@
+import myActivitiesReducer from '../../src/reducers/myActivitiesReducer';
+import {
+  MY_ACTIVITIES_GET_REQUEST,
+  MY_ACTIVITIES_GET_SUCCESS,
+  MY_ACTIVITIES_GET_FAILURE,
+} from '../../src/actions/constants';
+import activities from '../../src/fixtures/activities';
+
+describe('myActivitiesReducer', () => {
+  const initialState = {
+    requesting: false,
+    failed: false,
+    activities: [],
+  };
+
+  it('should return the initial state', () => {
+    expect(myActivitiesReducer(undefined, {})).toEqual({
+      requesting: false,
+      failed: false,
+      activities: [],
```

Comment:
You can re-use the defined `initialState` object, they pretty much the same

Reply:
Done